specification/network/resource-manager/Microsoft.Network/stable/2019-02-01/bastionHost.json

```diff
@@ -0,0 +1,440 @@
+                      "$ref":"#/definitions/BastionHostListResult"
+                   }
+                }
+             },
+             "x-ms-examples":{  
+                "List all Bastion Hosts for a given resource group":{  
+                   "$ref":"./examples/BastionHostListByResourceGroup.json"
+                }
+             },
+             "x-ms-pageable":{  
+                "nextLinkName":"nextLink"
+             }
+          }
+       },
+       "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Network/{virtualNetworkName}/bastionHosts/default":{  
+          "get":{  
+             "tags":[  
+                "BastionHosts"
+             ],
+             "operationId":"BastionHosts_ListVnet",
+             "description":"Gets the Bastion Host associated with a virtual network.",
+             "parameters":[  
+                {
+                    "name": "resourceGroupName",
+                    "in": "path",
+                    "required": true,
+                    "type": "string",
+                    "description": "The name of the resource group."
+                },
+                {
+                    "name": "virtualNetworkName",
+                    "in": "path",
+                    "required": true,
+                    "type": "string",
+                    "description": "The name of the virtual network."
+                },
+                {  
+                   "$ref":"./network.json#/parameters/ApiVersionParameter"
+                },
+                {  
+                   "$ref":"./network.json#/parameters/SubscriptionIdParameter"
+                }
+             ],
+             "responses":{  
+                "200":{  
+                    "description":"Success. The operation returns BastionHost associated with network.",
+                    "schema":{  
+                     "$ref":"#/definitions/BastionHost"
+                    }
+                 }
+             },
+             "x-ms-examples":{  
+                "Get Bastion Host for a given vnet":{  
+                   "$ref":"./examples/BastionHostListByVnet.json"
+                }
+             }
+          }
+       }
+    },
+    "definitions":{  
+       "BastionHostIPConfigurationPropertiesFormat":{  
+          "properties":{  
+             "subnet":{
+                "$ref":"./network.json#/definitions/SubResource",
+                "description":"Reference of the subnet resource. This field is a mandatory input."
+             },
+             "publicIPAddress": {
+                "$ref": "./network.json#/definitions/SubResource",
+                "description": "Reference of the PublicIP resource. This field is a mandatory input."
+             },
+             "provisioningState":{
+               "description":"The provisioning state of the resource.",
+               "$ref":"./network.json#/definitions/ProvisioningState"
+             },
+             "privateIPAllocationMethod": {
```

Comment:
Please reference definition from network.json to avoid duplicate code
```json
"privateIPAllocationMethod": {
  "$ref": "./network.json#/definitions/IPAllocationMethod",
  "description": "Private IP allocation method."
}
```

Reply:
done